e2e-tests/smart-update/run

```diff
@@ -10,7 +10,8 @@ API='pxc.percona.com/v9-9-9'
 TARGET_IMAGE_PXC=${IMAGE_PXC}
 CLUSTER="smart-update"
 CLUSTER_SIZE=3
-PROXY_SIZE=1
+PROXYSQL_SIZE=2
+HAPROXY_SIZE=1
```

Comment:
why we need two variables? minimal size is `2` for both

Reply:
it's not implemented in this PR, but we can add [K8SPXC-556](https://jira.percona.com/browse/K8SPXC-556) here